### ISSUE TITLE
new wife show's as mother of kinds

### ISSUE THREAD

**Issue by erdoking** [stuporglue/TreeTrumpet#21]
my uncle married new a short time ago. The new woman appears as the mother. Maybe there is an good idea to show all wives (if they has children together).

At the moment just the current wife is displayed...

**Comment by stuporglue** [comment]
Yes, this is one of several problems with the current pedigree viewer. 

I've been looking at several alternative pedigree viewers, but haven't found a suitable one which is also free to include in a project like this. 

At the moment the only way to find the alternate spouse is to search the name list and click the other wife's name. That should make her the focus.
